README.md

```diff
@@ -7,8 +7,5 @@ Welcome to [OpenLayers 3](http://openlayers.org/)!
 ## Examples
 Check out the [hosted examples](http://openlayers.org/en/master/examples/), the [workshop](http://openlayers.org/ol3-workshop/) or poke around the evolving [API docs](http://openlayers.org/en/master/apidoc/).
 
-## Questions
```

Comment:
I removed it from the README.md and added it to the website and CONTRIBUTING.md instead. The reason is that when you create an issue, you have a clickable link to CONTRIBUTING.md. And when you're looking for documentation, you'll first go to the website and not to the github repo.